File: src/main/java/com/teamwizardry/wizardry/api/capability/mana/IManaCapability.java
package com.teamwizardry.wizardry.api.capability.mana;

import net.minecraftforge.common.capabilities.ICapabilityProvider;

public interface IManaCapability extends ICapabilityProvider {

	long getMana();

	void setMana(long mana);

	long getMaxMana();

	void setMaxMana(long maxMana);

	long getBurnout();

	void setBurnout(long burnout);

	long getMaxBurnout();

	void setMaxBurnout(long maxBurnout);
}

File: src/main/java/com/teamwizardry/wizardry/api/capability/mana/IManaCapability.kt
package com.teamwizardry.wizardry.api.capability.mana;

public interface IManaCapability {
	double getMana();

	void setMana(double mana);

	double getMaxMana();

	void setMaxMana(double maxMana);

	double getBurnout();

	void setBurnout(double burnout);

	double getMaxBurnout();

	void setMaxBurnout(double maxBurnout);
}
